src/components/inputs/InlineField.tsx

```diff
@@ -180,7 +184,11 @@ export default function InlineField({
             value={value}
             {...inputProps}
             keyboardType={
-              android ? 'visible-password' : inputProps?.keyboardType
+              android
+                ? shouldFormatText
+                  ? 'default'
+                  : 'visible-password'
+                : inputProps?.keyboardType
```

Comment:
Maybe make this into a small helper function, to avoid nestig ternary operator in JSX? In helper it would be great to have it with if statements for better readability